assets/lokomotive-kubernetes/bootkube/assets.tf

```diff
@@ -33,6 +33,8 @@ resource "local_file" "kube-apiserver" {
     aggregation_ca_cert     = var.enable_aggregation == true ? base64encode(join(" ", tls_self_signed_cert.aggregation-ca.*.cert_pem)) : ""
     aggregation_client_cert = var.enable_aggregation == true ? base64encode(join(" ", tls_locally_signed_cert.aggregation-client.*.cert_pem)) : ""
     aggregation_client_key  = var.enable_aggregation == true ? base64encode(join(" ", tls_private_key.aggregation-client.*.private_key_pem)) : ""
+    replicas                = length(var.etcd_servers)
+    expose_on_all_ports     = var.expose_on_all_ports
```

Comment:
>This variable, also, is not changing just that (where to listen). Is changing the ip and the port it will listen too (for the HAProxy tricks)

I'd argue about that. For external API access, port 6443 is always used. I'd say swap of in-cluster port from 6443 to 7443 is an implementation detail, at least for now.

>I'd go with a better variable name or maybe separate the concepts: one var to expose on all interfaces and another to choose the port to listen (i.e. two vars instead of one doing this black magic not obvious at all from the name)

Hm, I'm not sure why we would need 2 parameters, if we don't plan on changing them?

Reply:
Well, an implementation detail that the cluster admin needs to be aware of. You can't manage a cluster that you don't know the components and how it is supposed to be working, IMHO.

And surely, we are changing them.

It's okay if you prefer to have one variable only for that too, IMHO. But it should be more clear what this does (and I guess some documentation in the chart variables would be awesome)

Reply:
It's now renamed to `expose_on_all_interfaces`. Is it sufficient? Can we mark it as resolved?

Reply:
Sorry, I missed this. Let's close it!